modules/core/src/lib/deck.ts

```diff
@@ -28,7 +28,7 @@ import Tooltip from './tooltip';
 import log from '../utils/log';
 import {deepEqual} from '../utils/deep-equal';
 import typedArrayManager from '../utils/typed-array-manager';
-import deckGlobal from './init';
+import {VERSION} from './init';
```

Comment:
Nit: Moving this probably makes the initialization code run later, but it is presumably OK in this case. But one has to be careful with such global registration stuff, deck global initialization and loader registration...